### PR TITLE
Spelling ruby

### PR DESCRIPTION
**Pull request by jsoref**
follow-up to #10743 

per @aibaars, I'm including the changes for the handful of shared items in this PR.

### CODE REVIEW

File: ruby/ql/lib/codeql/ruby/security/BadTagFilterQuery.qll [hunk 1]
```diff
@@ -65,7 +65,7 @@ predicate isBadRegexpFilter(HtmlMatchingRegExp regexp, string msg) {
   regexp.matches("<!-- foo --!>") and
   exists(int a, int b | a != b |
     regexp.fillsCaptureGroup("<!-- foo -->", a) and
-    // <!-- foo --> might be ambigously parsed (matching both capture groups), and that is ok here.
+    // <!-- foo --> might be ambiguous parsed (matching both capture groups), and that is ok here.
```
Comment (aibaars) [inline ruby/ql/lib/codeql/ruby/security/BadTagFilterQuery.qll:68]: This change would be grammatically incorrect. 

```suggestion
    // <!-- foo --> might be ambiguously parsed (matching both capture groups), and that is ok here.
```
or perhaps
```suggestion
    // <!-- foo --> might be parsed ambiguously (matching both capture groups), and that is ok here.
```

**Review by aibaars — COMMENTED** [review]
Thanks a lot for the changes. I think there is one grammatical mistake, but otherwise fine.

**Comment by aibaars** [comment]
@jsoref to fix the errors in the `synchronized files`  job you can run:
```
python3 config/sync-files.py --latest
```
or if that doesn't do the right thing for some reason, you can run
```
python3 config/sync-files.py ./ruby/ql/lib/codeql/ruby/controlflow/internal/ControlFlowGraphImplShared.qll
python3 config/sync-files.py ./ruby/ql/lib/codeql/ruby/security/BadTagFilterQuery.qll
python3 config/sync-files.py ./ruby/ql/lib/codeql/ruby/security/regexp/ExponentialBackTracking.qll
python3 config/sync-files.py ./ruby/ql/lib/codeql/ruby/security/regexp/RegexpMatching.qll
python3 config/sync-files.py ./ruby/ql/lib/codeql/ruby/security/regexp/SuperlinearBackTracking.qll
```

**Comment by aibaars** [comment]
The QL test error can be fixed by updating the expected output in `ruby/ql/test/query-tests/experimental/manually-check-http-verb/ManuallyCheckHttpVerb.expected` and replace `mappting => mapping`

File: ruby/ql/lib/codeql/ruby/security/BadTagFilterQuery.qll (hunk 1 above)
Comment (jsoref) [reply ruby/ql/lib/codeql/ruby/security/BadTagFilterQuery.qll:68]: Sigh. I saw that in one place and I know I switched it to the `ly` form.

Comment (aibaars) [reply ruby/ql/lib/codeql/ruby/security/BadTagFilterQuery.qll:68]: This file is shared with javascript and python, so you might have seen it there.

**Review by geoffw0 — APPROVED** [review]
Swift :+1:

**Review by aibaars — APPROVED** [review]

**Pull request merged**